components/datetime/src/format/input.rs

```diff
@@ -27,6 +27,8 @@ use crate::input::*;
 #[derive(Debug, Copy, Clone, Default)]
 #[non_exhaustive]
 pub struct DateTimeInputUnchecked {
+    /// The calendar kind, used for calendar checking only.
+    pub(crate) calendar_kind: Option<AnyCalendarKind>,
```

Comment:
thought: not a fan of making this even bigger, but it's fine. it's probably all optimized away

Reply:
This is an ephemeral type. I haven't been optimizing for size.

Reply:
Yep, I agree, was just flagging.